sympy/core/power.py

```diff
@@ -1669,12 +1671,14 @@ def _eval_as_leading_term(self, x, logx=None, cdir=0):
                     # Normally, f**e would evaluate to exp(e*log(f)) but on branch cuts
                     # an other value is expected through the following computation
                     # exp(e*(log(f) - 2*pi*I)) == f**e*exp(-2*e*pi*I) == f**e*(-1)**(-2*e).
-                    return self.func(f, e) * (-1)**(-2*e)
+                    return lt * (-1)**(-2*e)
                 elif im(ndir).is_zero:
                     log_leadterm = log(b)._eval_as_leading_term(x, logx=logx, cdir=cdir)
                     if log_leadterm.is_infinite is False:
-                        return exp(e*log_leadterm)
-            return self.func(f, e)
+                        return exp(e*log_leadterm)._eval_as_leading_term(x, logx=logx, cdir=cdir)
```

Comment:
I don't get this change. Could you explain or give an example as to why this has been done.

Reply:
Where do you see this change coming into play !